specification/sql/resource-manager/Microsoft.Sql/preview/2021-05-01-preview/ServerTrustCertificates.json

```diff
@@ -0,0 +1,297 @@
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Create server trust certificate.": {
+            "$ref": "./examples/ServerTrustCertificatesCreate.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ServerTrustCertificates"
+        ],
+        "description": "Deletes a server trust certificate that was uploaded from box to Sql Managed Instance.",
+        "operationId": "ServerTrustCertificates_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "name": "certificateName",
+            "in": "path",
+            "description": "Name of of the certificate to delete.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Certificate successfully deleted."
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 MissingCertificateName - The certificate name is missing.\n\n * 404 ResourceNotFound - The requested resource was not found.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 OperationIdNotFound - The operation with Id does not exist.\n\n * 409 OperationCancelled - The operation has been cancelled by user.\n\n * 409 OperationInterrupted - The operation on the resource could not be completed because it was interrupted by another operation on the same resource.\n\n * 500 OperationTimedOut - The operation timed out and automatically rolled back. Please retry the operation."
+          },
+          "202": {
+            "description": "Accepted request for deleting server trust certificate."
+          },
+          "204": {
+            "description": "Certificate resource not found."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Delete server trust certificate.": {
+            "$ref": "./examples/ServerTrustCertificatesDelete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/serverTrustCertificates": {
+      "get": {
+        "tags": [
+          "ServerTrustCertificates"
+        ],
+        "description": "Gets a list of server trust certificates that were uploaded from box to the given Sql Managed Instance.",
```

Comment:
'from box' seems a bit irrelevant, you could drop it. The important point to clarify would be what these certificates mean to the server.

Reply:
I would change this to:
"Gets a list of the server trust certificates which are used for secure communication between SQL On-Prem instance and the given Sql Managed Instance"

Reply:
I will directly add the description in the swagger PR to pass the review.
@nemijato will send out a PR to fix it in SQL repo as upstream?

Reply:
done